api/models/Burritos.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+var BurritoSchema = new Schema({
+  brandId: [{ type: Schema.Types.ObjectId, ref: 'Brand' }],
+  locationId: [{ type: Schema.Types.ObjectId, ref: 'Location' }],
```

Comment:
See above.

Reply:
Getting rid of the array for `locationId`, but couldn't 1 brand's burrito be available at different locations? Or are we treating each burrito unique to location instead of just unique to each brand?

Reply:
I'm not sure, whichever way you think is most correct

Reply:
Eh let's treat them unique by location for now and see how that works. We could also use these to average out a rating across all locations or something.